README.md

```diff
@@ -76,6 +76,8 @@ In the root folder, run:
 pnpm --filter frontend dev
 
 # Start the backend (localhost:3000)
+pnpm --filter backend exec prisma generate
+pnpm --filter backend exec prisma migrate deploy
 pnpm --filter backend dev
```

Comment:
this section only has `pnpm dev` in `CONTRIBUTING.md`, not sure which is more "correct" but with `pnpm dev` the frontend didn't continue running for me